helpers/demo/announce-test.js

```diff
@@ -11,21 +11,24 @@ class AnnounceTest extends LitElement {
 				display: block;
 				width: 400px;
 			}
-			d2l-button {
-				margin: 10px 10px 0 0;
+			d2l-input-text {
+				margin-bottom: 10px;
 			}
 		`;
 	}
 
 	render() {
 		return html`
-			<d2l-input-text type="text" value="I like cookies." aria-label="message to announce"></d2l-input-text>
+			<d2l-input-text id="msg1" type="text" value="I like cookies but I also like donuts and many other really yummy things." aria-label="first message to announce"></d2l-input-text>
+			<d2l-input-text id="msg2" type="text" value="I also like cake." aria-label="second message to announce"></d2l-input-text>
 			<d2l-button @click=${this._handleAnnounce} type="button">Announce</d2l-button>`;
 	}
 
 	_handleAnnounce() {
-		const value = this.shadowRoot.querySelector('d2l-input-text').value;
-		announce(value);
+		const value1 = this.shadowRoot.querySelector('#msg1').value;
```

Comment:
I updated the demo so that it's possible to demo/test the case where more than one announcement is made in succession.